package.json

```diff
@@ -76,7 +76,7 @@
     "ts-node": "^3.3.0",
     "tslint": "^5.7.0",
     "tslint-test-config-non-relative": "file:test/external/tslint-test-config-non-relative",
-    "typescript": "~2.5.1"
+    "typescript": "^2.6.0-dev.20171014"
```

Comment:
This is not ready to be merged. We need to wait for the next official typescript release.
2.6.0-rc cannot be used because https://github.com/Microsoft/TypeScript/issues/19164 breaks the formatter tests